src/wp-includes/js/media/controllers/edit-image.js

```diff
@@ -32,21 +32,36 @@ EditImage = wp.media.controller.State.extend(/** @lends wp.media.controller.Edit
 	},
 
 	/**
+	 * @summary Activates a frame for editing a featured image.
+	 *
 	 * @since 3.9.0
+	 *
+	 * @returns {void}
 	 */
 	activate: function() {
 		this.frame.on( 'toolbar:render:edit-image', _.bind( this.toolbar, this ) );
 	},
 
 	/**
+	 * @summary Deactivates a frame for editing a featured image.
+	 *
 	 * @since 3.9.0
+	 *
+	 * @returns {void}
 	 */
 	deactivate: function() {
 		this.frame.off( 'toolbar:render:edit-image' );
 	},
 
 	/**
+	 * @summary Adds a toolbar with a back button.
+	 *
+	 * When the back button is pressed it checks whether there is a previous state. In case there is a previous state it
```

Comment:
Punctuation: `When the back button is pressed, it checks whether there is a previous state. In case there is a previous state, it sets that previous state. Otherwise it closes the frame.`

Reply:
Also: please check line length